txscript/scriptnum.go

```diff
@@ -8,9 +8,13 @@ const (
 	maxInt32 = 1<<31 - 1
 	minInt32 = -1 << 31
 
-	// maxScriptNumLen is the maximum number of bytes data being interpreted
-	// as an integer may be.
-	maxScriptNumLen = 4
+	// maxScriptNumLen is the maximum number of bytes
+	// data being interpreted as an integer may be.
+	maxScriptNumLen = 9
```

Comment:
There is no need for this.  As discussed below where it is used, it would break consensus.